lib/client-bundle-plugin.js

```diff
@@ -137,10 +137,24 @@ export default function (inputOptions, outputOptions, resolveFileUrl) {
       const dependencies = getDependencies(clientOutput, clientEntry);
 
       if (property.startsWith(allSrcPlaceholder)) {
-        const depCodes = dependencies.map(
-          (name) => clientOutput.find((item) => item.fileName === name).code,
-        );
-        return JSON.stringify([clientEntry.code, ...depCodes].join(';'));
+        const allModules = [
+          clientEntry,
+          ...dependencies.map((name) =>
+            clientOutput.find((item) => item.fileName === name),
+          ),
+        ];
+
+        const inlineDefines = [
+          ...allModules.map(
+            (item) =>
+              `self.nextDefineUri=location.origin+${resolveFileUrl(item)};${
```

Comment:
I know we aren't governed by https://url.spec.whatwg.org/#goals, but maybe worth considering:

> Standardize on the term URL. URI and IRI are just confusing. In practice a single algorithm is used for both so keeping them distinct is not helping anyone.

Reply:
> Hmm, good catch. Maybe it's not correctly defined then, let me recheck.

Hah, right, it's actually something I correctly handled first, but then reverted because app seemed to work fine anyway. Fixed.

Reply:
I guess this makes sense; however the right place to change it would be upstream in https://github.com/surma/rollup-plugin-off-main-thread/blob/master/loader.ejs first, as this name comes from there (and I want to keep Squoosh loader not _too_ different from upstream one to make future updates easier).